CONTRIBUTING.rst

```diff
@@ -69,192 +69,9 @@ Ready to contribute? Here's how to set up `pyfar` for local development using th
 
 7. Submit a pull request on the develop branch through the GitHub website.
 
-Pull Request Guidelines
------------------------
+Guide lines
+-----------
 
-Before you submit a pull request, check that it meets these guidelines:
+You can find our general guidelines in the `pyfar community`_.
```

Comment:
maybe we can add a general intro at the beginning of this doc. something that explains the general and specific guidelines. Maybe this part can be moved to the beginning so it is clear from the start.